package.json

```diff
@@ -42,7 +43,6 @@
     "@zerollup/ts-transform-paths": "^1.7.11",
     "case-sensitive-paths-webpack-plugin": "^2.3.0",
     "cross-env": "^7.0.0",
-    "cz-conventional-changelog": "3.3.0",
```

Comment:
Were we using this one? Do we need a replacement?

Reply:
It's being used via npx